research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,153 @@
+> | `disabled`     | `boolean`                                    | Specifies whether a component should be disabled.                                          |  |
+
+## Behaviors
+
+> Define the behaviors of the component including user interactions and expected screen reader behaviors. If there are also considerations for different screen sizes or form factors, include those as well. Link the relevant [W3 ARIA specification](https://www.w3.org/TR/WCAG21) (and read it).
+>
+> - How should disabled behavior work (are things focusable)?
+> - How and when should focus indicators show up?
+
+### Keyboard interaction
+
+> Define the keyboarding behaviors of the component. What specific keyboard events should be observed and how they affect the component?
+>
+> - Consider arrow behaviors vs tabbing
+> - Do modifiers affect things? (shift, ctrl, meta)
+> - Are states changed as a result of a keystroke?
+> - Are some events triggered on key down vs key press?
+> - RTL settings?
+>
+> Example:
+>
+> | Key       | Description                                                                                                                                                    |
+> | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+> | Up/Right  | Increments the value of the {ComponentName} by the amount specified by the `step` prop. If the `shift` modifier is pressed, increases by 10x the `step` value, clamped within `min`/`max` values. |
+> | Down/Left | Decrements the value of the {ComponentName} by the amount specified by the `step` prop. If the `shift` modifier is pressed, decreases by 10x the `step` value, clamped within `min`/`max` values. |
+
+### Mouse interaction
+
+> Define the mouse behaviors of the component.
+>
+> - What specific mouse events (down, up, move) are being listened to and how does that affect the component?
+> - For behaviors which are accessible through mouse input, can you also do the same thing using a keyboard?
+>
+> Example:
+>
+> - `mousedown` should immediately attempt to set value to the appropriate value.
+> - `mousemove` should be attached to the window during mousedown to track window-wise move events. Value should be updated appropriately. Note; if a `mousemove` occurs
+>   without the primary button pressed, tracking should cancel and treat the even as a `mouseup`. (Edge case: mouse down, move cursor out of window, release mouse)
+> - `mouseup` should remove the `mousemove` event.
+
+### Touch interaction
+
+> Define the touch behaviors of the component.
+>
+> - Are there different behaviors than in mouse scenarios?
+> - Are touch targets large enough for fat fingers? (44x44px is the recommended minimum size.) For example, the `thumb` of a slider might appear to be small, but the touchable area should be larger.
+
+### Screen reader accessibility
```

Comment:
I understand this starts getting into some design, but I think we should possibly have a default recommendation for system colors of the anatomy for `forced-color` support. I think this deserves its own issue to discuss so don't block the PR on this.